www/js/diary/cards/TripCard.tsx

```diff
@@ -46,7 +46,7 @@ const TripCard = ({ trip }) => {
     navigation.navigate("label.details", { tripId: trip._id.$oid });
   }
 
-  const mapOpts = { zoomControl: false, dragging: false };
+  const mapOpts = { minZoom: 4, maxZoom: 17, dragging: false };
```

Comment:
Please revert this change.

We only want zooming to be enabled in the details view. We don't want zooming enabled on the `TripCard`s.